pages/about.tsx

```diff
@@ -68,98 +69,131 @@ const AboutPage = () => {
+          usually increments by 1 byte for each opcode, the{' '}
+          <RelativeLink to="#60" title="PUSHx" /> instruction is longer than a
+          single byte, and causes the PC to skip their parameter. The{' '}
+          <RelativeLink to="#56" title="JUMP" /> instruction does not increase
+          the PC's value, instead, it modifies the program counter to a position
+          specified by the top of the stack.{' '}
+          <RelativeLink to="#57" title="JUMPI" /> does this as well, if its
+          condition is true (a nonzero code value), otherwise it increments the
+          PC like other instructions.
         </p>
       </SectionWrapper>
 
-      <SectionWrapper header={<H3>The stack</H3>} anchorKey="stack">
+      <SectionWrapper header={<H3>The Stack</H3>} anchorKey="stack">
         <p className="pb-8">
-          The stack is a list of 32-byte elements. The stack is used to put the
-          parameters that are needed by the instructions, and their result
-          values. When a new value is put on the stack, it is put on top, and
-          only the top values are used by the instructions. The stack currently
-          has a maximum limit of 1024 values. All instructions interact with the
-          stack, but it can be directly manipulated with instructions like{' '}
-          <RelativeLink to="#60" title="PUSH1" />,{' '}
+          The stack is a list of 32-byte elements used to store smart contract
+          instruction inputs and outputs. The stack is only accessible during
+          smart contract execution. When a new value is put on the stack, it is
+          put on top, and only the top values are used by the instructions. The
+          stack currently has a maximum limit of 1024 values. All instructions
+          interact with the stack, but it can be directly manipulated with
+          instructions like <RelativeLink to="#60" title="PUSH1" />,{' '}
           <RelativeLink to="#50" title="POP" />,{' '}
           <RelativeLink to="#80" title="DUP1" />, or{' '}
           <RelativeLink to="#90" title="SWAP1" />.
         </p>
       </SectionWrapper>
 
-      <SectionWrapper header={<H3>The memory</H3>} anchorKey="memory">
+      <SectionWrapper header={<H3>The Memory</H3>} anchorKey="memory">
         <p className="pb-8">
-          The memory is a region that only exists during the smart contract
-          execution, and is accessed with a byte offset. While all the 32-byte
-          address space is available and initialized to 0, the size is counted
-          with the highest address that was accessed. It is generally read and
-          written with <RelativeLink to="#51" title="MLOAD" /> and{' '}
-          <RelativeLink to="#52" title="MSTORE" /> instructions, but is also
-          used by other instructions like{' '}
+          Unlike EVM code, EVM memory is only accessible during smart contract
+          execution. Memory is not persistent, and ceases at the end of the call
```

Comment:
May be my English, but just "ceases" is unclear to me. "is destroyed" would be more clear.

Reply:
iirc we said ceases because "destroyed" was technically incorrect, however we had it previously

Reply:
changed to is destroyed